Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.9-slim
```

Comment:
It looks like the development environment is using 3.9. @bemoody has asked whether we can match this to the version used in tests (3.7)?

Reply:
Sure, it's a good idea to bump the python version in the future, though